### PR TITLE
add all-in-one yaml and helper script

### PR DESCRIPTION
**Pull request by dmesser**
add a helper script that takes all manifests generated by helm and concatenates them into a single YAML for simple direct installation using `kubectl -f https://raw.githubusercontent.com/dmesser/operator-lifecycle-manager/quickstart/deploy/upstream/quickstart/olm.yaml`

### CODE REVIEW

File: deploy/upstream/values.yaml [hunk 1]
```diff
@@ -7,21 +7,21 @@ writeStatusName: '""'
 olm:
   replicaCount: 1
   image:
-    ref: quay.io/coreos/olm@sha256:995a181839f301585a0e115c083619b6d73812c58a8444d7b13b8e407010325f
+    ref: quay.io/openshift/origin-operator-lifecycle-manager:latest
```
Comment (alecmerdler) [inline deploy/upstream/values.yaml]: We probably want Alpine images for upstream.

Comment (njhale) [reply deploy/upstream/values.yaml]: do we have any built?

**Pull request merged**